auth-gateway/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "auth-gateway",
+  "version": "0.2.0",
```

Comment:
any particular reason for starting the auth-gateway project at version 0.2?

Reply:
This should probably just be 0.2.9, or an independent version. I think that versioning packages separately from those that may depend on them is confusing, so prefer single version across all projects. 

Since we're not using Lerna to manage JS stuff anymore, we'll have to manage these version liens in each package.json file manually, or I guess write something to manage their individual publication that replicates Lerna's features...or not publish them to npm.

Reply:
We don't have a single version across all sub projects and I don't think we should. There might be backwards incompatible changes to, e.g., cloudtools (which will be made a subproject soon, currently at https://github.com/Nealelab/cloudtools), that have no corresponding backwards incompatible changes to hail.

I do not have strong feelings about auth-gateway's initial version since, AFAICT, this package is not intended to pushed to a public package repository and consumed by non-hail-team projects/users.

I do not believe we should increment the auth-gateway's version each time we increment the hail pip version. Almost all the changes that go into hail's python package will be independent of auth-gateway.